interview/password.py

```diff
@@ -13,7 +13,45 @@ def generate_password() -> str:
 
     # HINT you will require the use of a random number generator for this function
     # https://docs.python.org/3/library/random.html#random.randint
-    
+
     # TODO implement generate_password function
+    import random 
+
+    password = ''
+
+    # relationship between random and randit 
+    alphabet = 'abcdefghijklmkopqrstuvwxyz'
+    r= alphabet[random.randint(0, len(alphabet) -1)]
```

Comment:
this is good -- my only criticism is that when using random this way, you always append the same 5 letters (ex: aaaaa ...)